typings/index.d.ts

```diff
@@ -1621,6 +1621,7 @@ declare module 'discord.js' {
     public type: 'voice';
     public userLimit: number;
     public join(): Promise<VoiceConnection>;
+    public createVoiceAdapter(): object;
```

Comment:
would it be possible to import the type from `@discordjs/voice` if it is installed?

Reply:
There's no conditional type imports. We can import it directly and it'll default to `any` if not present (and you use `skipLibCheck` in your tsconfig)

Reply:
> There's no conditional type imports. We can import it directly and it'll default to `any` if not present (and you use `skipLibCheck` in your tsconfig)

This is already the case with `@types/ws` being a dev dep. I see no issue doing the same with `@discordjs/voice`?